lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -46,8 +46,9 @@
         &:hover,
         &:focus,
         &.active {
-          color: $uxpl-blue-hover-active;
-          border-bottom-color: $uxpl-blue-hover-active;
+          color: rgba(0,117,180, 1);
+          border-bottom-color: rgba(0,117,180, 1);
```

Comment:
Please define a new variable that equate to `rgba(0,117,180, 1)`

Reply:
Also, no need for the 1, make it rgb(0, 117, 180)